app/components/billing/payment.js

```diff
@@ -78,13 +78,26 @@ export default Component.extend({
 
   updatePlan: task(function* () {
     if (this.selectedPlan.isFree) {
+      this.metrics.trackEvent({
+        action: 'Plan Reverted to Free Plan',
+        category: 'Subscription',
+      });
```

Comment:
Can users cancel in the modal that this action opens? If yes, then this data may become inaccurate.

Reply:
You're right, thank you!